lvjit/LuaFunctionBuilder.cpp

```diff
@@ -1070,166 +1070,86 @@ bool Lua::FunctionBuilder::do_newtable(TR::BytecodeBuilder* builder, Instruction
    return true;
 }
 
-bool Lua::FunctionBuilder::do_add(TR::BytecodeBuilder* builder, Instruction instruction) {
+bool Lua::FunctionBuilder::do_math(TR::BytecodeBuilder* builder, Instruction instruction) {
    // rb = RKB(i);
    // rc = RKC(i);
-   builder->Store("rb", jit_RK(builder, GETARG_B(instruction)));
-   builder->Store("rc", jit_RK(builder, GETARG_C(instruction)));
+   TR::IlValue *rb = jit_RK(builder, GETARG_B(instruction));
+   TR::IlValue *rc = jit_RK(builder, GETARG_C(instruction));
 
    // if (ttisinteger(rb) && ttisinteger(rc))
-   auto isrbint = jit_checktag(builder, builder->Load("rb"), LUA_TNUMINT);
-   auto isrcint = jit_checktag(builder, builder->Load("rc"), LUA_TNUMINT);
+   auto isrbint = jit_checktag(builder, rb, LUA_TNUMINT);
+   auto isrcint = jit_checktag(builder, rc, LUA_TNUMINT);
 
    auto addints = builder->OrphanBuilder();
```

Comment:
Since the operation is not necessarily ADD, I think `addints` should be renamed to something more general like `ints`.